core/client/src/main/java/alluxio/client/file/options/SetAttributeOptions.java

```diff
@@ -28,9 +31,12 @@
 @PublicApi
 @NotThreadSafe
 public final class SetAttributeOptions {
+  private final Logger mLogger = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
```

Comment:
@aaudiber any suggestions here? it would be great if we could use "LOG" consistently